app/__init__.py

```diff
@@ -81,13 +83,25 @@ def create_app(application):
     logging.init_app(application, statsd_client)
     firetext_client.init_app(application, statsd_client=statsd_client)
     mmg_client.init_app(application, statsd_client=statsd_client)
+
     aws_ses_client.init_app(application.config['AWS_REGION'], statsd_client=statsd_client)
+    aws_ses_stub_client.init_app(
+        application.config['AWS_REGION'],
+        statsd_client=statsd_client,
+        stub_url=os.environ.get('SES_STUB_URL')
+    )
+    # If a stub url is provided for SES, then use the stub client rather than the real SES boto client
+    if os.environ.get('SES_STUB_URL') and notify_environment not in ['live', 'production']:
```

Comment:
you should use `application.config['SES_STUB_URL']` rather than `os.environ` - just to keep with the pattern that we only fetch from `os.environ` in config.py.

also, we could hard code `SES_STUB_URL = None` in the `Live` config, that way we wouldn't need this gross `notify_environment` check as even if it was set, it would be overwritten by class inheritance in prod.